naming convention

Tearran opened the pull request:
setting up for a category subcategory loop array. 

defined some early prefixes to reference when needed.  
https://github.com/armbian/configng/wiki/naming_convention

set up a manner to run directly from the file or from the config.sh to assist with testing. 
```bash
bash ~/lib/config/network_configure_wireless.sh" network_configure_wirerless::set_wpa_connect beta 12345678
```
alternitively from config
```
config.sh" foo  network_configure_wirerless::set_wpa_connect beta 1245678
```
will result with a file in "~/.local/etc/wpa_supplicant/wpa_supplicant.conf" currently
```
network={
        ssid="beta"
        #psk="12345678"
        psk=cbc0b65ba2ceaa9158cc47bca151b0643ff0ab3611a26a341fb278b99ab9f1ec
}
```
Grep Sed has been tested to omit the #psk. since Unsure of wpa_supplicant mechanics was not included.





set a manner to pipe a output to a menu on jampi-config.

igorpecovnik commented on bin/config:
```diff
@@ -111,20 +131,27 @@ check_opts() {
     if [ "$found" == false ]; then
-      echo "Invalid function name"
-      see_help_dev
+      echo "Invalid Gorup Option"
```
Group?

Tearran replied:
Meant it to be `Function group` 

Perhaps 'Options' may better suited?

igorpecovnik replied:
Just noticed typo gorup vs group ... its fine.

Tearran commented:
some place holders removed, 
renamed jampi-config to a seemly appropriate descriptive name tui (“tee-you-eye”)
Added 
-  tui (T.U.I.) 
    - menu example [git wiki tui](https://github.com/Tearran/configng/wiki/tui)
    - menu desktop serch example [/lib/config/access_install_desktops.sh](https://github.com/Tearran/configng/blob/main/lib/config/access_install_desktops.sh)
    
- config 
   - help message  [git wiki Naming-Convention](https://github.com/Tearran/configng/wiki/Naming-Convention)
        - -h basic help message
        - -l list of available cli commands I.E. cli help list
        - -r run a group from the cli list
    - Reformatted  cli interface to better alline  with [git wiki Naming-Convention](https://github.com/Tearran/configng/wiki/Naming-Convention)
    - changed foo to -r 
    - changed `config foo group::function` to `config -r group function` 
    - Example ` ./bin/config -r benchymark see_boot_times time ` 



bash utilities has not been used.

Tearran commented:
- [x] needed name conventions
 - [x] base md wiki  
- [x] Use /etc/os-release convention `VERSION_CODENAME` [[AR-978]](https://armbian.atlassian.net/browse/AR-978?focusedCommentId=11951) 
- [x] practical example [build.contrib.sh](https://github.com/Tearran/configng/raw/main/build.contrib.sh) armbian-bookworm-utils-config.0.1.deb
 - [ ] move onto #4
